MatrixKit/Controllers/MXKRoomViewController.m

```diff
@@ -380,6 +380,9 @@ - (void)viewDidAppear:(BOOL)animated
     {
         hasAppearedOnce = YES;
     }
+    
+    //  Mark all messages as read when the room is displayed
+    [self.roomDataSource.room.summary markAllAsRead];
```

Comment:
This is too brutal. It will reset the read marker as well.

Reply:
It seems we need a new method in MXRoomSummary to reset local unread counts without changing the read marker.

Reply:
Updated to use `markAllAsReadLocally` method.